### PR TITLE
Add price change with colors to stonks

### LINKED ISSUE #8: [feature] Colors on stonks based on price
**Is your feature request related to a problem? Please describe.**
The current output of stonks/quote is boring. It could use some color.

**Describe the solution you'd like**
I propose that Current price have some color based on if its positive or negative from yesterday. The algorithm is as follows:
if currentPrice == yesterdaysPrice then,
  use white color for current price,
else if currentPrice < yesterdaysPrice then,
  use red color for current price,
else,
  use green color for current price. 

**Describe alternatives you've considered**
Alternatively, we could use colored triangles to denote upwards or downward price shift.
▲ - U+25B2 BLACK UP-POINTING TRIANGLE
▼ - U+25BC BLACK DOWN-POINTING TRIANGLE

This may be more friendly to colorblind people since the color will be a secondary meaning on top of the arrow.

**Additional context**
Thank you for your time!

### PR DESCRIPTION
**Pull request by mynameisivan**
Resolves #8

### CODE REVIEW

File: commands/stonks.js [hunk 1]
```diff
@@ -18,14 +18,29 @@ exports.run = async (client, message, args, level) => {
       if(Object.keys(data).length === 0) {
         message.reply(`Invalid ticker [${company}]`);
       } else {
-        message.channel.send(`\`\`\`\n
+        const priceDiffPercent = Math.round((data.c - data.pc) / data.pc * 10000)/100;
```
Comment (mynameisivan) [inline commands/stonks.js:21]: Theres gotta be a better way to do this right?

File: commands/stonks.js [hunk 2]
```diff
@@ -18,14 +18,29 @@ exports.run = async (client, message, args, level) => {
...
+        let pricePrefix = '\+';
+        if(priceDiffPercent < 0) {
+          arrowDirection = '\u25BC'; //down 
+          pricePrefix = '\-';
+        } 
+
+        // important to have price prefix match with diff markdown colors
+        // + in the beginning of the line for green
+        // - for red
+        message.channel.send(`\`\`\`diff
```
Comment (mynameisivan) [inline commands/stonks.js:34]: Turns out you can't do normal text coloring but its only using markdown "hacking". 

Here I'm leveraging diff styling to get +green and -red colors. It only highlights when the line starts with +/-

Comment (mynameisivan) [reply commands/stonks.js:34]: Sample output:
image

Comment (I-AM-STEVEe) [reply commands/stonks.js:34]: the green is not as green as the red is red, you get what I mean?

File: commands/stonks.js (hunk 1 above)
Comment (mynameisivan) [reply commands/stonks.js]: Using toLocaleString recommended by @I-AM-STEVEe 👍

File: commands/stonks.js (hunk 2 above)
Comment (mynameisivan) [reply commands/stonks.js:39]: Ya but I can't even choose the colors :(

File: commands/stonks.js [hunk 3]
```diff
@@ -18,14 +20,32 @@ exports.run = async (client, message, args, level) => {
       if(Object.keys(data).length === 0) {
         message.reply(`Invalid ticker [${company}]`);
       } else {
-        message.channel.send(`\`\`\`\n
+        const percentDiff = Math.abs(data.c - data.pc) / data.pc * 100;
```
Comment (mynameisivan) [inline commands/stonks.js:23]: Duplicate code, will abstract.

File: commands/stonks.js (hunk 2 above)
Comment (I-AM-STEVEe) [reply commands/stonks.js:39]: where are the colors defined?

Comment (mynameisivan) [reply commands/stonks.js:39]: https://gist.github.com/matthewzring/9f7bbfd102003963f9be7dbcf7d40e51#syntax-highlighting

Comment (BrandonWLee) [reply commands/stonks.js:39]: I get what you mean

**Comment by BrandonWLee** [comment]
I can't approve after merge? aight.
I approve

File: commands/stonks.js (hunk 2 above)
Comment (mynameisivan) [reply commands/stonks.js:39]: They don't think it be like it is but it do.

**Pull request merged**